packages/aila/src/protocol/jsonPatchProtocol.ts

```diff
@@ -167,6 +259,28 @@ export const PatchDocumentSchema = z.object({
   value: JsonPatchValueSchema,
 });
 
+// This is the schema that we send when requesting Structured Outputs
+// From the LLM. We are limited to not having min, max and we also
+// have constraints about ensuring each object has a type attribute
+// as its first attribute in the definition.
+// Note, that unless/until we migrate all past messages we won't be
+// able to use this to validate the patches currently in the database
+// because they would be missing a "type" attribute.
```

Comment:
This is frustrating, but maybe not avoidable. We just need to be careful when updating the types

Maybe there's a future version where can wrap the current types with the LLM types, but it sounds like the better approach is to migrate the current data